tests/integration_test.py

```diff
@@ -20,15 +20,19 @@ def tmpdir():
     finally:
         shutil.rmtree(dir_, ignore_errors=True)
 
+DEFAULT_SOURCE_PIPELINE = 'videotestsrc is-live=true'
 
-def pulsevideo_cmdline():
+
+def pulsevideo_cmdline(source_pipeline=None):
+    if source_pipeline is None:
+        source_pipeline = DEFAULT_SOURCE_PIPELINE
```

Comment:
Python strings are immutable aren't they? So you don't need to do the "default argument = None` dance for this.